be/src/runtime/runtime_state.h

```diff
@@ -113,6 +113,8 @@ class RuntimeState {
     ObjectPool* global_obj_pool() const;
     void set_query_ctx(pipeline::QueryContext* ctx) { _query_ctx = ctx; }
     pipeline::QueryContext* query_ctx() { return _query_ctx; }
+    pipeline::FragmentContext* fragment_ctx() { return _fragment_ctx; }
+    void set_fragment_ctx(pipeline::FragmentContext* fragment_ctx) { _fragment_ctx = fragment_ctx; }
```

Comment:
check null?

Reply:
Needn't, `set_fragment_ctx` is just a set method, it's is ok even though `fragment_ctx` is nullptr.